source/developers/quickstarts/first_openedx_pr.rst

```diff
@@ -266,6 +257,19 @@ Now push your changes to a new branch in your fork::
 
     git push --set-upstream origin developer_quickstart
 
+If you get a ``fatal: Authentication failed`` error, run::
+
+   gh auth login
+
+and answer the questions:
+
+::
+
+   ? What account do you want to log into? GitHub.com
+   ? What is your preferred protocol for Git operations? HTTPS
+   ? Authenticate Git with your GitHub credentials? Yes
+   ? How would you like to authenticate GitHub CLI? Login with a web browser
```

Comment:
There are a few ways to login to the CLI and I wouldn't want to represent that you have to log in with the web browser. In fact I use SSH for Git. Maybe linking to https://cli.github.com/manual/gh_auth_login and/or https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/about-authentication-to-github would help people understand their options and choose what works for them.

Reply:
good call, will update